src/API/API.php

```diff
@@ -55,6 +55,10 @@ public function load_endpoints() {
 		// Load recent donations endpoint
 		$recentDonations = new Endpoints\Reports\RecentDonations();
 		$recentDonations->init();
+
+		// Load income endpoint
+		$income = new Endpoints\Reports\Income();
+		$income->init();
```

Comment:
Remember at some point to use a FQCN array of the endpoints so you can loop through them and initialize.

Reply:
Just set that up! ref: https://github.com/impress-org/givewp/pull/4437/commits/cd58a137c6d495b83ade716182ae958522c36882